### PR TITLE
Remove scaling at hover over the button to exit debug mode

### PR DESCRIPTION
**Pull request by THEb0nny**
Remove the scaling of the button to exit debug mode when hover.

https://github.com/microsoft/pxt/issues/9684

### CODE REVIEW

**Comment by THEb0nny** [comment]
@kimprice you can view?

File: theme/themes/pxt/collections/menu.overrides [hunk 1]
```diff
@@ -33,10 +33,6 @@
     background: @invertedBackground;
 }
 
-#mainmenu > .menu > .ui.item:hover > .icon {
-    transform: scale(1.2);
-}
```
Comment (kimprice) [inline theme/themes/pxt/collections/menu.overrides:38]: This change is going to impact more than just the exit debug button. I think we'll need to do something targeted for ".exit-debugmode-btn". Possibly using the "not" css attribute.

Comment (THEb0nny) [reply theme/themes/pxt/collections/menu.overrides:38]: I did it using the not selector, I don’t know why I didn’t do it right away...

**Review by kimprice — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**